lib/exchange_api_web/controllers/order_controller.ex

```diff
@@ -5,58 +5,52 @@ defmodule ExchangeApiWeb.OrderController do
 
   def index_open(conn, %{"ticker" => ticker}) do
     with {:ok, tick} <- get_ticker(ticker) do
-      {status, data} = Exchange.open_orders(tick)
-      json(conn, %{status: status, data: data})
+      set_http_status(conn, Exchange.open_orders(tick))
     end
   end
```

Comment:
Instead of having `set_http_status` deal with `:error` responses, you can have your `FallbackController` deal with them and only deal with the `:ok` responses here. One good way to do this is adding your call to `Exchange.open_orders/1` to your `with` statement.